addons/hr_expense/static/src/views/list.js

```diff
@@ -44,11 +44,30 @@ export class ExpenseListController extends ListController {
         return this.is_account_invoicing && records.length && records.every(record => record.data.state === 'approve') && this.isExpenseSheet;
     }
 
+    displayPayment() {
+        const records = this.model.root.selection;
+        return this.is_account_invoicing && records.length && records.every(record => record.data.state === 'post' && record.data.payment_state === 'not_paid') && this.isExpenseSheet;
+    }
+
     async onClick (action) {
         const records = this.model.root.selection;
         const recordIds = records.map((a) => a.resId);
         const model = this.model.rootParams.resModel;
-        await this.orm.call(model, action, [recordIds]);
+        let res = await this.orm.call(model, action, [recordIds]);
```

Comment:
As you will not reassign on res, you can change `let` for `const` and please name it as `resultAction` or something similar :sweat_smile: 
Btw, we have checked together that sometimes, functions called `action_....` don't return an action. It will be cool to fix that in a future PR. Thank you!

Reply:
I will take it in https://github.com/odoo/odoo/pull/110518